app/src/test/java/dev/arkbuilders/navigator/data/utils/PathExtKtTest.kt

```diff
@@ -0,0 +1,88 @@
+package dev.arkbuilders.navigator.data.utils
+
+import io.mockk.every
+import io.mockk.mockk
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Assertions.assertThrows
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import org.junit.runner.RunWith
+import org.mockito.junit.MockitoJUnitRunner
+import java.nio.file.Path
+import java.nio.file.Paths
+import kotlin.io.path.name
+import kotlin.io.path.notExists
+
+@RunWith(MockitoJUnitRunner::class)
+class PathExtKtTest {
+
+    private lateinit var testee: Path
+
+    @BeforeEach
+    fun setUp() {
+        testee = Paths.get("PATH")
+    }
+
+    @Test
+    fun givenPath_whenFindNotExistCopyNameFalse_thenReturnNewPath() {
+        val sourcePath = mockk<Path>()
+        val destPath = mockk<Path>()
+
+        every { destPath.fileName } returns Paths.get("PATH")
+        every { sourcePath.resolve(Paths.get("PATH")) } returns Paths.get("PATH")
+        every { sourcePath.resolve(Paths.get("PATH")).notExists() } returns false
+
+        every { sourcePath.resolve("PATH_1.") } returns Paths.get("PATH_1.")
```

Comment:
You can test without a mock, use `createTempFile`

Reply:
Could you please elaborate this usage? Currently the tested method is interacting with `Path` objects. And createTempFile from File class returns File object instead.

Reply:
`createTempFile` returns `Path`

Reply:
It looks like it does not work. I would like to keep it as is. I also update result from the method execution

Reply:
What exactly doesn't work?

Reply:
> What exactly doesn't work?

It works now. Thanks. Nice suggestion